distributed/client.py

```diff
@@ -3170,8 +3170,7 @@ def _graph_to_futures(
             self._send_to_scheduler(
                 {
                     "op": "update-graph",
-                    "graph_header": header,
-                    "graph_frames": frames,
+                    "graph": Serialized(header, frames),
```

Comment:
The important change is not to move from `ToPickle` to `Serialize` but rather to wrap the header+frames in the corresponding container. I suspected that Serialize is faster but haven't verified. For objects that properly implement pickle5 this shouldn't matter really.

Reply:
I suggest adding an in-line comment explaining this to keep our future selves from removing this "excessively explicit" serialization step.